src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindOneIntegrationTest.java

```diff
@@ -354,6 +354,31 @@ public void inConditionNonArrayArray() {
           .body("errors[0].errorCode", is("INVALID_FILTER_EXPRESSION"));
     }
 
+    @Test
+    public void ninConditionNonArrayArray() {
+      String json =
+          """
+            {
+              "findOne": {
+                "filter" : {"_id" : {"$nin": false}}
```

Comment:
In our discussion we decided not going to use allow filtering and so won't support $nin for _id

Reply:
No
for _id $nin, we will use something like this
select * from [purchase_database.my](http://purchase_database.my/)_vsearch where query_text_values[‘_id’] != ‘3’ and query_text_values[‘_id’] != ‘4’;
that is what we discussed.